server/repository/src/migrations/v1_01_12/barcode_sync.rs

```diff
@@ -0,0 +1,13 @@
+use crate::{migrations::sql, StorageConnection};
+
+pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
+    sql!(
+        connection,
+        r#"
+        ALTER TABLE barcode ADD is_sync_update bool NOT NULL DEFAULT False;
+        ALTER TABLE public.barcode RENAME COLUMN value TO gtin;
```

Comment:
We also need to re-juggle the triggers, as per commit mentioned in issue: https://github.com/openmsupply/open-msupply/commit/5b41b5bdfddcbbca42edb898f2ca27a118c9f0db

Reply:
like this? 69f6c510c

Reply:
🙏

Just doing cargo test and manual test